pkg/coscheduling/coscheduling.go

```diff
@@ -205,18 +205,18 @@ func (cs *Coscheduling) Permit(ctx context.Context, state *framework.CycleState,
 	podGroupName := pgInfo.name
 	minAvailable := pgInfo.minAvailable
 	bound := cs.calculateBoundPods(podGroupName, namespace)
-	waiting := cs.calculateWaitingPods(podGroupName, namespace)
-	current := bound + waiting + 1
+	// The bound is calculated from the snapshot. The current pod does not exist in the snapshot during this scheduling cycle.
+	current := bound + 1
 
```

Comment:
After we change `running` to `bound`, the pods in bound and waiting maybe duplication.